api/config.py

```diff
@@ -212,6 +223,8 @@ def initialize_db():
         upsert=True
     )
 
+    log_db.access_log.create_index('origin.id')
```

Comment:
I actually talked to Ryan about not indexing anything but the `_id` as Mongo will most likely only be a temporary holder for the log messages.